janus_client/Cargo.toml

```diff
@@ -13,7 +13,8 @@ rust-version = "1.63"
 backoff = { version = "0.4.0", features = ["tokio"] }
 derivative = "2.2.0"
 http = "0.2.8"
-janus_core = { version = "0.1", path = "../janus_core" }
+janus_core = { path = "../janus_core" }
+janus_messages = { path = "../janus_messages" }
```

Comment:
We will need versions for both of these dependencies before we next publish janus_client

Reply:
Oops, I knew there must have been a good reason that was there! I also added this to `janus_core`'s dep on `janus_messages`.